src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,49 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("numbers size는 3이다.")
```

Comment:
```suggestion
    @DisplayName("Set은 중복된 데이터를 허용하지 않는다.")
```

메서드의 동작보다는 검증하고 싶은 내용을 DisplayName에 담아보면 어떨까요?

Reply:
넵, 피드백 감사합니다 !!